docs/source/conceptual/visual_language_models.md

```diff
@@ -0,0 +1,224 @@
+# Vision Language Models (VLM)
+
+## What is VLM?
+
+Visual Language Model (VLM) are models that consume both visual and textual inputs to generate text.
+
+These models are trained on multimodal data, which includes both images and text.
+
+VLMs can be used for a variety of tasks, such as image captioning, visual question answering, and more.
+
+<div class="flex justify-center">
+    <pre>placeholder for architecture diagram</pre>
+</div>
+
+With VLM, you can generate text from an image. For example, you can generate a caption for an image, answer questions about an image, or generate a description of an image.
```

Comment:
```suggestion
A vision language model can generate text from an image. For example, you can generate a caption for an image, answer questions about an image, or generate a description of an image.
```
these two sentences are already what you said above

Reply:
removed